skeleton-typescript-webpack/package.json

```diff
@@ -113,6 +113,7 @@
     "@types/webpack": "^2.0.0",
     "@types/whatwg-fetch": "^0.0.32",
     "aurelia-protractor-plugin": "^1.0.1",
+    "babel-polyfill": "^6.16.0",
```

Comment:
That's so odd. Which Node.js version are you on? Shouldn't it have a Promise implementation? In any case, we could use Bluebird instead of Babel, since it's already there as a dependency (no need for us to have multiple Promise implementations).